### PR TITLE
fix: send ordinal routes, closes #4444

### LINKED ISSUE #4444: Unable to send Ordinals, confirmation screen does not appear
I was trying to preview the "send" flow for sending an Ordinal but encountered a bug.

I am able to input the Send To address and choose my fee, but instead of displaying a confirmation dialog, I am sent to a blank modal.

Video:

https://github.com/leather-wallet/extension/assets/122313287/468a96ce-9a79-4555-b97a-1ab100468363

### LINKED ISSUE #4467: chore: add better integration tests for send ordinal
The send ordinal flow was [broken critically ](https://github.com/leather-wallet/extension/issues/4444)but we didn't know it as we lacked test coverage there. 

This task involves fixing this and making sure we have tests to cover the send flow

We may need to mock an ordinal or else have a test one available to send so we can enter the flow

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6669290341).

This PR fixes some introduced errors with send ordinal routing. 

I got the routes fixed but sending the actual ordinal errors as you can see here: 

https://github.com/leather-wallet/extension/assets/2938440/93c79fa4-4ac4-4118-9707-cf14ecfc469b

### CODE REVIEW

File: src/app/pages/send/ordinal-inscription/hooks/use-send-inscription-form.tsx [hunk 1]
```diff
@@ -100,17 +103,20 @@ export function useSendInscriptionForm() {
...
-          fee: feeValue,
-          inscription,
-          utxo,
-          recipient: values.recipient,
-          time,
-          feeRowValue,
-          tx: hex,
-        },
-      });
+      return navigate(
```
Comment (pete-watters) [inline src/app/pages/send/ordinal-inscription/hooks/use-send-inscription-form.tsx:106]: I need to look into this more as if I don't compose the route like this I get a weird error when trying to send

Screenshot 2023-10-27 at 16 24 47

**Review by kyranjamie — APPROVED** [review]
Curious if this can also be fixed when changing routes with `relativeTo` prop? Or we have to have the `/` prefixed path?

**Comment by kyranjamie** [comment]
Merging already for release

**Comment by pete-watters** [comment]
> Curious if this can also be fixed when changing routes with `relativeTo` prop? Or we have to have the `/` prefixed path?

I tried that and it wasn't working. I can check a cleaner solution, I just wanted to get a fix out ASAP

**Pull request merged**